src/Home.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import Slider from './Slider/Slider.js'
-import Movies from './Allmovies/AllMovies'
+import Movies from './Allmovies/allmovies.js'
```

Comment:
Will be better to use the same approach of naming files and folders in all project by all team members. (for example: components & services with upper letter CamelCase and for folders with lower letter dash-case.